packages/react-native/src/primitives/Radio/getRadioDimensions.ts

```diff
@@ -16,23 +21,32 @@ import { RadioDimensions, RadioStyles, Size } from './types';
  * (e.g., 'small' => { radioContainerSmall, radioDotSmall } )
  */
 
+export const RADIO_DOT_PROPORTION = 0.6;
+
 export const getRadioDimensions = (
   size: Size,
   styles: RadioStyles
 ): {
-  radioContainerSize: RadioDimensions;
-  radioDotSize: RadioDimensions;
+  radioContainerDimensions: RadioContainerDimensions;
+  radioDotDimensions: RadioDotDimensions;
 } => {
   if (typeof size === 'number') {
     return {
-      radioContainerSize: { height: size, width: size },
-      radioDotSize: { height: size * 0.5, width: size * 0.5 },
+      radioContainerDimensions: {
+        borderWidth: Math.floor(size / 10) - 1,
```

Comment:
Called this out in the thread, but i think the `borderWidth` should be stable across our `Size` variants. It is very easy for users to make changes if they choose to

Reply:
See https://github.com/aws-amplify/amplify-ui/pull/2913#discussion_r1017099842